src/PhpCs/FiveLab/Sniffs/Commenting/AnnotationsSniff.php

```diff
@@ -68,5 +65,13 @@ private function processAnnotation(File $phpcsFile, int $lineNumber, string $ann
                 );
             }
         }
+
+        if (\str_contains($value, '[]')) {
+            $phpcsFile->addErrorOnLine(
+                'Please use vector type annotation for arrays.',
+                $lineNumber,
+                ErrorCodes::PROHIBITED
```

Comment:
We must add another error code, because on some cases we want to ignore only our error (array as vector).

As example we ignore few error in our `src` and `tests` folder.